lib/ui/screens/login_screen.dart

```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
 }
 
 class _LoginScreenState extends State<LoginScreen> {
-  FocusNode _passwordFocusNode;
-
+  FocusNode _passwordFocusNode, _emailFocusNode;
+  bool isSignUp;
```

Comment:
Please use enum page {signUp, signIn} instead of bool

Reply:
Ok

Reply:
what is the problem with using bool

Reply:
if isSignUp == true we know page is signUp, but if isSignUp == false it tells us nothing what page should be. enum tells there are only 2 pages and tells what are these pages

Reply:
isSignUp == false simply means login here

Reply:
We know the logic we want to implement so it's obvious for us. But if someone else takes a look into this code it is not so obvious.

Reply:
I dont know how to use enum in dart

Reply:
Okay, I'll take care of this enum myself